packages/google_maps_flutter/lib/src/bitmap.dart

```diff
@@ -58,8 +58,9 @@ class BitmapDescriptor {
     String assetName, {
     AssetBundle bundle,
     String package,
+    bool mipmaps = true,
```

Comment:
Please also add dart doc to explain this param.

Reply:
You would need to give me an example. I don't see any param docs in this .dart file.

Reply:
Something like:
///
/// Set `mipmaps` to ... to allow ...., `mipmaps` is ... by default.